exporter/datadogexporter/traces_exporter.go

```diff
@@ -33,6 +34,13 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/scrub"
 )
 
+var traceCustomHTTPFeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"exporter.datadogexporter.TraceAgentUseCustomHTTPClient",
```

Comment:
I don't think 'trace agent' is a meaningful concept for end-users of the Datadog exporter: it's an implementation detail. Is there any way to rephrase this to make it more meaningful to end-users?

Reply:
Renamed to `TraceExport...`